tests/baselines/reference/docker/office-ui-fabric.log

```diff
@@ -106,6 +115,168 @@ Standard output:
+@fluentui/docs:   Types of property 'design' are incompatible.
+@fluentui/docs:     Type 'Requireable<InferProps<{ position: Requireable<string>; display: Requireable<string>; top: Requireable<string>; right: Requireable<string>; bottom: Requireable<...>; ... 16 more ...; maxHeight: Requireable<...>; }>>' is not assignable to type 'Validator<ComponentDesignProp>'.
+@fluentui/docs: ERROR in /office-ui-fabric-react/packages/fluentui/react-northstar/src/components/List/List.tsx
+@fluentui/docs: ERROR in /office-ui-fabric-react/packages/fluentui/react-northstar/src/components/List/List.tsx(186,1):
+@fluentui/docs: TS2322: Type '{ debug: PropTypes.Requireable<boolean>; items: (props: Record<string, any>, propName: string, componentName: string, ...args: any[]) => any; selectable: (props: Record<string, any>, propName: string, componentName: string, ...args: any[]) => any; ... 16 more ...; accessibility: PropTypes.Requireable<...>; }' is not assignable to type 'WeakValidationMap<WithAsProp<ListProps>>'.
+@fluentui/docs:   Types of property 'design' are incompatible.
+@fluentui/docs:     Type 'Requireable<InferProps<{ position: Requireable<string>; display: Requireable<string>; top: Requireable<string>; right: Requireable<string>; bottom: Requireable<...>; ... 16 more ...; maxHeight: Requireable<...>; }>>' is not assignable to type 'Validator<ComponentDesignProp>'.
+@fluentui/docs: ERROR in /office-ui-fabric-react/packages/fluentui/react-northstar/src/components/List/ListItem.tsx
+@fluentui/docs: ERROR in /office-ui-fabric-react/packages/fluentui/react-northstar/src/components/List/ListItem.tsx(244,1):
+@fluentui/docs: TS2322: Type '{ contentMedia: PropTypes.Requireable<any>; content: PropTypes.Requireable<any>; debug: PropTypes.Requireable<boolean>; header: PropTypes.Requireable<any>; ... 18 more ...; accessibility: PropTypes.Requireable<...>; }' is not assignable to type 'WeakValidationMap<ListItemProps & { as?: any; } & { index: number; }>'.
+@fluentui/docs:   Types of property 'design' are incompatible.
+@fluentui/docs:     Type 'Requireable<InferProps<{ position: Requireable<string>; display: Requireable<string>; top: Requireable<string>; right: Requireable<string>; bottom: Requireable<...>; ... 16 more ...; maxHeight: Requireable<...>; }>>' is not assignable to type 'Validator<ComponentDesignProp>'.
+@fluentui/docs: ERROR in /office-ui-fabric-react/packages/fluentui/react-northstar/src/components/Provider/Provider.tsx
+@fluentui/docs: ERROR in /office-ui-fabric-react/packages/fluentui/react-northstar/src/components/Provider/Provider.tsx(210,3):
+@fluentui/docs: TS2322: Type 'Requireable<InferProps<{ siteVariables: Requireable<object>; componentVariables: Requireable<object>; componentStyles: Requireable<object>; fontFaces: Requireable<InferProps<...>[]>; staticStyles: Requireable<...>; animations: Requireable<...>; }>>' is not assignable to type 'Validator<ThemeInput<any>>'.
```

Comment:
this is the other unique error; haven't figured out what is going on here

Reply:
`object` is no longer assignable to `Record<string, ComponentSlotStyleInput<{}, {}>>` (because the target has an index signature and the source doesn't -- the Record's value type isn't important.)

This is also correct as far as I can tell, but also disappears with the build from March 2.